Dependency/peg-markdown-highlight/HGMarkdownHighlighter.m

```diff
@@ -258,7 +258,9 @@ - (void) clearHighlightingForRange:(NSRange)range
 	[textStorage applyFontTraits:_clearFontTraitMask range:range];
 	[textStorage removeAttribute:NSBackgroundColorAttributeName range:range];
 	[textStorage removeAttribute:NSLinkAttributeName range:range];
-    if (self.targetTextView.typingAttributes && self.resetTypingAttributes)
+    if (self.targetTextView.typingAttributes
+        && self.resetTypingAttributes
+        && self.defaultTypingAttributes[NSParagraphStyleAttributeName] != NULL)
```

Comment:
the `!= NULL` part is not needed; just `&& self.defaultTypingAttributes[NSParagraphStyleAttributeName]` is enough (and technically it should be `nil` not `NULL`).